README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
-- 🔹 **Private sales:** In a few cases, you may be able to sell private company stock to another private party, such as an accredited investor who wants to become an investor in the company, but this is fairly rare. This is often called the **secondary market**. Sales generally require the agreement and cooperation of the company. For example, typically your shares would be subject to a **right of first refusal** in favor of the company (meaning you couldn’t sell your shares to a third party without offering to sell it to the company first). Another possible roadblock is that private buyers may want the company's internal financials to establish the value of the stock, and this typically requires the cooperation of the company. There have been some efforts such as [SharesPost](http://sharespost.com/), [Equidate](https://www.equidateinc.com/), and [EquityZen](https://equityzen.com/) to establish a market around such sales, particularly for well-known pre-IPO companies, but it’s still not a routine practice. Quora has [more discussion](https://www.quora.com/How-do-employees-in-startups-sell-stock-in-the-secondary-markets) on this topic.
+We’ve now seen how complex it is to assess the value of equity in private companies. It is even more complex than this, because the value is always changing.
 
-### Kinds of stock
+- In order to finance a company, companies **fundraise** by selling shares in their business to [investors](http://www.investopedia.com/terms/i/investor.asp) in exchange for capital. With the focus on rapid growth in startups, fundraising (usually from investors called **venture capitalists**) is the most likely path for these companies early on. (More on the [stages of startups](#stages-of-a-startup) is below.)
+- As these companies add (or “issue”) stock, the outstanding shares goes up, and the percentage ownership of each shareholder goes down. This is called [**dilution**](http://www.investopedia.com/terms/d/dilution.asp), but don’t let that word freak you out.
+  - Dilution doesn’t necessarily mean that as a shareholder you’re losing anything-usually, as a company raises money and issues stock, the hope and intention is that what you do have is worth more. The size of your slice gets relatively smaller, but if the company is growing, the size of the cake gets bigger.
+	- For example, a typical startup might have three rounds of funding, with each round of funding issuing 20% more shares. At the end of the three rounds, there are more outstanding shares-roughly 70% more in this case, since 120%×120%×120% is 173%-and each shareholder owns proportionally less of the company.
+- On the other hand, the [**valuation**](https://en.wikipedia.org/wiki/Valuation_(finance)) of the company, which is the present value investors believe the company has, can change over time as well. If the company is doing well, making more revenue or gaining users (indicating future revenue), the company’s valuation will usually be going up.
```

Comment:
I think it's worth tying valuation here to the "price" of a share. The higher a company's valuation, the more money it costs for someone to buy a share of the company. This might *seem* obvious, but to the 100% naive, I think this could be helpful to call out.

Reply:
I agree--and the section on 409A is too much of a gloss considering how important it is to understand that connection.

Reply:
✔️